source/channels/message-priority.rst

```diff
@@ -0,0 +1,46 @@
+Set message priority (beta)
```

Comment:
Can we try to answer the question of "when should I use a message priority label"? ie we don't want people to abuse it. So providing some example use cases for the priority labels and ack might be useful in the docs